paywalls/src/androidMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/Paywall.kt

```diff
@@ -1,4 +1,4 @@
-package io.shortway.kobankat.ui.revenuecatui
+package com.revenuecat.purchases.kmp.ui.revenuecatui
```

Comment:
We should rename this module from `paywalls` to `revenuecatui` or something like that, since we might eventually add more UI other than paywalls here?

Reply:
`revenuecatui` can work yes! I named it `ui` at first, like in purchases-android, but got various conflicts. Don't exactly remember what it was, but maybe Xcode didn't like it. 

Of course the Gradle module name and published artifact ID don't have to be the same, so we can still publish this module as artifact `:ui`, and change the Gradle module name when we build more UI stuff?

Reply:
Yeah I think that's ok for now 👍

Reply:
So I tried to name it `ui` again to check if this was a fluke, but it really doesn't work haha. I'm getting these errors related to Compose Multiplatform on iOS:

>Cannot access class 'androidx.compose.ui.Modifier'. Check your module classpath for missing or conflicting dependencies. 
  
Since these are part of the `compose.ui` package, maybe it conflicts on the `ui` part there? Pretty weird.